Azure/azure-cli
# [AppService] BREAKING CHANGE: az webapp list-runtimes: Add Dotnet6 support and update runtimes

**Description**
Fixes #17950 
- Add .NET 6 support for AppService webapps. Runtime dotnet|6.0 can be used with az webapp create and az webapp up
- Values are hardcoded, since stacks API is in 2020-12-01 api version

Update runtimes: 
- Remove `php|7.2`, `java|1.8|tomcat|8.0`, `java|11|tomcat|8.0`, `java|7|tomcat 7 , 8.5, 9, SE` for windows 
- Add `node|14-lts` for windows
- Remove `php|7.2` for linux 
- Add java JBOSSEAP for linux 

**Testing Guide**
- az webapp list-runtimes (--linux) should show dotnet|6.0 as an available runtime
- az webapp up run in a .NET6 app should work and create a dotnet|6.0 webapp
- az webapp create --runtime "dotnet|6.0" should create a dotnet|6.0 webapp

---

This checklist is used to make sure that common guidelines for a pull request are followed.
